src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+(() => {
+  const body = document.querySelector('body');
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+  });
```

Comment:
This is a recommendation. You are preventing the default context menu from appearing on right-click. While this is not necessarily a problem, it can be confusing for users who expect the default behavior. If you need to prevent the default context menu, it would be better to do it only when it's necessary for your application's functionality.